### ISSUE TITLE
Add well-known commands

### ISSUE THREAD

**Issue by DanielOaks** [DanielOaks/modern-irc#4]
These are commands that are generally accepted and supported on IRC daemons. So long as we have notes that these commands MAY NOT be supported by servers, should be fine to include them.

* [ ] `KLINE`
* [ ] `UNKLINE`
* [ ] `DLINE`
* [ ] `UNDLINE`
* [ ] `SA*` -- `SAMODE`, `SANICK`, etc etc. SA may stand for Service Admin? If so, these are probably safe to simply ignore without any issues? Eh, investigate and see what servers do, I think these may be loadable modules in most IRCds that do support them anyway, rather than built-in proper

**Comment by ailin-nemui** [comment]
WHOX isn't a command, it signals that extended WHO usage is possible

**Comment by DanielOaks** [comment]
Yep, that's my mistake. I've been trying to document WHOX on a separate branch, but it's slow-going because it's really dodgy trying to mix-and-match them together into one spec.
